puzzles/templates/modals/edit_status.html

```diff
@@ -0,0 +1,41 @@
+<div class="modal fade" id="editstatus" tabindex="-1" role="dialog" aria-labelledby="edittitle" aria-hidden="true">
+    <div class="modal-dialog" role="document">
+        <div class="modal-content">
+            <div class="modal-header">
+                <h5 class="modal-title" id="edittitle">Edit Status of <span id='puzzletitle'></span></h5>
+                <button type="button" class="close" data-dismiss="modal" aria-label="Close">
+                    <span aria-hidden="true">&times;</span>
+                </button>
+            </div>
+            <form method="post" action="/puzzles/update_status/">
+                {% csrf_token %}
+                <div class="form-group">

```

Comment:
add modal-body to get padding here

Reply:
Ah, thanks, the modal looked weird (no edge padding) and I was wondering why